### PR TITLE
[docs] added details about import data file with S3

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/migrate/migrate-steps/#import-data-file

### CODE REVIEW

File: docs/content/preview/migrate/migrate-steps.md [hunk 1]
```diff
@@ -308,7 +308,14 @@ yb-voyager import data file --export-dir <EXPORT_DIR> \
         –-has-header
 ```
 
-Refer to [import data file](../reference/yb-voyager-cli/#import-data-file) for details about the arguments.
+Import data file also allows you to load directly from your data files stored on AWS S3. The S3 bucket URI can be provided to the `data-dir` flag as follows:
```
Comment (rahulb-yb) [inline docs/content/preview/migrate/migrate-steps.md:311]: Can we have a separate subsection under import data for import from S3? Similar to how we have a separate section for import data from file

**Review by rahulb-yb — COMMENTED** [review]
Left a comment

File: docs/content/preview/migrate/migrate-steps.md (hunk 1 above)
Comment (aishwarya24) [reply docs/content/preview/migrate/migrate-steps.md:315]: addressed it. PTAL @rahulb-yb

**Review by rahulb-yb — APPROVED** [review]
LGTM, thanks for the change!

File: docs/content/preview/migrate/reference/yb-voyager-cli.md [hunk 2]
```diff
@@ -336,7 +336,7 @@ The valid *arguments* for import data file are described in the following table:
 | Argument | Description/valid options |
 | :------- | :------------------------ |
 | [--batch-size](#batch-size) <number> | Size of batches generated for ingestion during [import data]. |
-| [--data-dir](#data-dir) <path> | Path to the directory containing the data files to import. |
+| [--data-dir](#data-dir) <path> | Path to the directory/S3 prefixed bucket URI containing the data files to import. |
```
Comment (ddhodge) [inline docs/content/preview/migrate/reference/yb-voyager-cli.md:339]:
```suggestion
| [--data-dir](#data-dir) <path> | Path to the directory or S3-prefixed URI of the bucket containing the data files to import. |
```

File: docs/content/preview/migrate/reference/yb-voyager-cli.md [hunk 3]
```diff
@@ -526,7 +526,9 @@ yb-voyager import data file .... \
 --data-dir s3://voyager-data
 ```
 
-Refer to [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) for more details.
+The authentication mechanism used when accessing the S3 bucket through yb-voyager is the same as using the AWS CLI tool to access the bucket.
```
Comment (ddhodge) [inline docs/content/preview/migrate/reference/yb-voyager-cli.md:529]:
```suggestion
The authentication mechanism for accessing an S3 bucket using yb-voyager is the same as that used by the AWS CLI.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**